src/questions/introduction_to_defi.yaml

```diff
@@ -0,0 +1,1147 @@
+  hint: Disadvantges of centralised
+  explanation: CeFi is not open source thus developers cannot access it. Complete control lies on the hans of few authorites thus no one can interfere with the code.
+  answerKeys:
+    - B
+    - C
+  subTopics:
+    - "centralised-vs-decentralised-finance"
+  difficultyLevel: "Medium"
+  choices:
+    - content: People do not trust centralised finance.
+      key: A
+      order: 0
+    - content: Developers are not allowed to build on top of them.
+      key: B
+      order: 1
+    - content: Complete control lies in the hand of a few authorities.
+      key: C
+      order: 2
+    - content: They will never get outdated.
+      key: D
+      order: 3
+- uuid: db6081e9-bc7a-4430-af57-cccd291dec2d
+  type: SingleChoice
+  content: Conversion of material assets into digital assets using DeFi is known as.....
+  hint: NoHint
+  explanation: Tokenization means conversion of material assets into digital tokens using DeFi and smart contracts.
+  answerKeys:
+    - B
+  subTopics:
+    - "uses-of-decentralised-finance"
+  difficultyLevel: "High"
+  choices:
+    - content: Digitization
+      key: A
+      order: 0
+    - content: Tokenization
+      key: B
+      order: 1
+    - content: Consensus Mechanism
+      key: C
+      order: 2
+    - content: None of the these
+      key: D
+      order: 3
+- uuid: 66f8098a-3b77-458f-a7f8-6c951bd19343
+  type: MultipleChoice
+  content: How does tokenization empower DeFi?
+  hint: Benefits of Tokenization
+  explanation: Tokens make it easy to trade material assets in the form of tokens. These tokens are extremely secured and cannot be stolen or hacked.
+  answerKeys:
+    - A
+    - B
+    - D
+  subTopics:
+    - "uses-of-decentralised-finance"
+  difficultyLevel: "High"
+  choices:
+    - content: Material assets could be converted.
+      key: A
+      order: 0
+    - content: Lending and borrowing become easy.
+      key: B
+      order: 1
+    - content: Forgery of assets increases.
+      key: C
+      order: 2
+    - content: Assets become more secured.
+      key: D
+      order: 3
+- uuid: 945a2feb-81db-46bb-8e33-9dcf95ce7307
+  type: SingleChoice
+  content:  What is the major issue involved in Smart Contracts?
+  hint: NoHint
+  explanation: Smart contracts are complicated currently. The tiniest flaw in their code can lead to loss funds.
+  answerKeys:
+    - C
+  subTopics:
+    - "uses-of-decentralised-finance"
+  difficultyLevel: "Medium"
+  choices:
+    - content: Smart contracts are not trustworthy.
+      key: A
+      order: 0
+    - content: It is difficult to code smart contracts.
+      key: B
+      order: 1
+    - content: The slightest flaw in code can lead to a loss of funds.
+      key: C
+      order: 2
+    - content: They are easy to hack.
+      key: D
+      order: 3
+- uuid: df4273d6-ea88-45b0-90f1-c28d496dbb6e
+  type: SingleChoice
+  content: What is the current number of transactions that Ethereum can process per second?
```

Comment:
https://blog.coinbase.com/scaling-ethereum-crypto-for-a-billion-users-715ce15afc0b. this blog says its 15

Reply:
Done